packages/ripple-binary-codec/package.json

```diff
@@ -22,8 +22,8 @@
   "scripts": {
     "build": "tsc -b && copyfiles ./src/enums/definitions.json ./dist/enums/",
     "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo",
-    "prepare": "npm run build && npm test",
-    "test": "jest",
+    "prepare-OLD": "npm run build && npm test",
```

Comment:
Do we no longer need this prepare script?

Reply:
Actually not sure who this was changed, I put it back